assets/json/menu.json

```diff
@@ -397,7 +444,7 @@
           "data": {
             "key": "submenubutton",
             "variables": {
-              "text": "Start",
+              "text": "START",
```

Comment:
Why is this capitalized now

Reply:
Figma design had the Button text in all caps. Was not sure if we are following Figma completely.

Reply:
Can keep as is, capital or not doesn't really matter too much